src/twisted/newsfragments/9235.feature

```diff
@@ -0,0 +1 @@
+t.w.c._HTTP11ClientFactory has a useful repr
```

Comment:
Please explain how this helps users.

Reply:
Also please don't abbreviate.  And since this is a private class, who cares?  What are the public consequences?